### DISCUSSION TITLE
Importing H@H archives folders to library

### DISCUSSION THREAD

**Discussion by teishahbc** [Difegue/LANraragi#1042] [Q&A] [answered]
Hi everyone,

I'm having some trouble with importing my downloaded images from H@H directly into my Lanraragi library. I've tried a few different methods, but nothing seems to be working.

Does anyone know how to do this directly, without having to manually making them into zip/rar/pdf? Also, is there a way to reset Lanraragi's statistics? I'd like to start fresh with my collection.

Any help would be greatly appreciated!

**Comment by Difegue** [comment, marked as answer]
There's no support for raw images currently, you'll need to pack them into an archive of some form.

As for resetting, the "Reset Database" option in settings should do it for you ?
